src/dsp_tools/cli.py

```diff
@@ -616,7 +616,7 @@ def run(args: list[str]) -> None:
         print("\nThe process was terminated because of an Error:")
         print(err.message)
         sys.exit(1)
-    except Exception as err:
+    except Exception as err:  # noqa: BLE001 (blind-except)
```

Comment:
I believe in this single case I think this makes sense. Because this is the outer most function of the CLI, we want to catch all exceptions that were not anticipated by us. Log them and tell the user to contact us.

Reply:
Yes exactly. I believe BLE001 is a rule that we generally want to be activated, but in this specific case, it is okay. We will never have to fix it. Due to the ignore comment, it doesn't appear as warning in the IDE.